.yarnrc.yml

```diff
@@ -3,6 +3,10 @@ nodeLinker: node-modules
 npmScopes:
   tensor-oss:
     npmAlwaysAuth: true
+  tensor-hq:
+    npmAlwaysAuth: true
+    npmAuthToken: "${GH_AUTH_TOKEN:-}"
+    npmRegistryServer: "https://npm.pkg.github.com"
```

Comment:
This is a public repo: why are we pulling from the private packages?

Are you sure you are pulling the right tensor-hq repos? Should be the ones on npm, NOT private packages.